docs/get-started/install.rst

```diff
@@ -39,20 +44,50 @@ Do the following instead.
     brew install opencv
     brew info openblas
     brew install openblas
-    git clone --recursive https://github.com/dmlc/mxnet
+    git clone --recursive -b engine https://github.com/dmlc/mxnet
     cd mxnet; cp make/osx.mk ./config.mk; make -j$(sysctl -n hw.ncpu)
 
 It turns out that installing ``openblas`` is necessary, in addition to modify the makefile, to fix `one of the build issues <https://github.com/dmlc/mxnet/issues/572>`_.
 
 Setup Python and its environment
 --------------------------------
 
-Refer to MXNet installation document for `Python package installation. <http://mxnet.readthedocs.io/en/latest/how_to/build.html>`_. One of the most common problem a beginner runs into is not setting the environment variable to tell Python where to find the library. Suppose you have installed ``mxnet`` under your home directory and is running bash shell. Put the following line in your ``~/.bashrc`` (or ``~/.bash_profile``)
+Refer to MXNet installation document for `Python package installation. <http://mxnet.io/get_started/ubuntu_setup.html#install-mxnet-for-python>`_. One of the most common problem a beginner runs into is not setting the environment variable to tell Python where to find the library. Suppose you have installed ``mxnet`` under your home directory and is running bash shell. Put the following line in your ``~/.bashrc`` (or ``~/.bash_profile``)
 
 ::
 
     export PYTHONPATH=~/mxnet/python:$PYTHONPATH
 
+If the installation meets Numpy or MXNet version conflicts with your other projects, we recommend using `virtualenv <https://virtualenv.pypa.io/en/stable/>`_ and `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_ to resolve the issue:
+
+::
+
+    sudo pip install virtualenv virtualenvwrapper
+
+Add three lines to your shell startup file (.bashrc, .profile, etc.) to set the location where the virtual environments should live, the location of your development project directories, and the location of the script installed with this package:
+
+::
+
+    export WORKON_HOME=$HOME/.virtualenvs
+    export PROJECT_HOME=$HOME/Devel
```

Comment:
In fact, as for me, the $PROJECT_HOME is unnecessary in most conditions. Maybe we can leave out this to simplify the configuration procedure.